core/src/main/scala/com/softwaremill/session/CsrfDirectives.scala

```diff
@@ -36,6 +40,9 @@ trait CsrfDirectives {
     }
   }
 
+  // for backward compatibility
+  def randomTokenCsrfProtection[T](checkMode: CsrfCheckMode[T]): Directive0 = hmacTokenCsrfProtection(checkMode)
```

Comment:
maybe we should add an `@deprecated` here

Reply:
Added a deprecated warning to both java and scala implementations.